lib/CodeInput.js

```diff
@@ -10,16 +10,19 @@ const DEFAULT_KEYBOARD_TYPE = Platform.select({
 export default class CodeInput extends PureComponent {
   static propTypes = {
     size: PropTypes.number.isRequired,
+    initialValue: PropTypes.string,
   };
 
   static defaultProps = {
     size: 4,
     onChange: () => {},
-    renderSeparator: () => null
+    renderSeparator: () => null,
   };
 
   state = {
-    code: new Array(this.props.size).fill(null),
+    code: this.props.initialValue
+      ? this.props.code.split('')
```

Comment:
`this.props.code`? I believe that this should be like `this.props.initialValue.substr(0, this.props.size).split('')`